app/controllers/quotes_controller.rb

```diff
@@ -1,14 +1,29 @@
 class QuotesController < ApplicationController
+  before_action :authenticate_user!
   def show
     @quote = Quote.find params[:id]
   end
 
+  def edit
+    @quote = current_user.quotes.find params[:id]
+  end
+
+  def update
+    @quote = current_user.quotes.find params[:id]
+    if @quote.update quote_params
+      redirect_to @quote
+    else
+      render :edit
+    end
+  end
+
   def new
     @quote = Quote.new
   end
 
   def create
     @quote = Quote.new quote_params
+    @quote.user = current_user
```

Comment:
That sets the association between the logged in user and the quote being created.  I don't know if there is a more standard practice way of creating it though.